docs/introduction/open-source/public_code.md

```diff
@@ -1,89 +1,93 @@
-A codebase MUST include all source code that the policy is based on. | | "yes" or "not applicable", do we consider it based on the API?
-All policy and source code that the codebase is based on MUST be documented, reusable and portable. | yes | code dependencies are OSI
+A codebase MUST include all source code that the policy is based on, unless used for fraud detection. |  | "yes" or "not applicable", do we consider it based on the API?
 Policy SHOULD be provided in machine readable and unambiguous formats. | yes | OpenAPI is in machine readable yaml format
 Continuous integration tests SHOULD validate that the source code and the policy are executed coherently. | yes | [GitHub workflow](https://github.com/open-zaak/open-zaak/blob/main/.github/workflows/ci.yml), [API Test Platform](https://api-test.nl/server/1/6b5fe675-694d-4948-8896-5eae88d30ef0/14bc91f7-7d8b-4bba-a020-a6c316655e65/latest/)
 
 ## [Create reusable and portable code](https://standard.publiccode.net/criteria/reusable-and-portable-codebases.html)
 
-- [x] designed to be reusable from the start
+- [ ] criterion met.
 
 Requirement | meets | links and notes
 -----|-----|-----
 The codebase MUST be developed to be reusable in different contexts. | yes | Designed to be so from the start.
 The codebase MUST be independent from any secret, undisclosed, proprietary or non-open licensed code or services for execution and understanding. | yes | installation supports docker, kubernetes, vmware appliances, bare-metal is possible
-The codebase SHOULD be in use by multiple parties. | yes | Deployed in multiple sandbox environments (e.g: Utrecht is testing it, others Den Haag, Delft looking at it.)
-The roadmap SHOULD be influenced by the needs of multiple parties. | yes | [Dimpact](https://www.dimpact.nl/openzaak), [market consultation](https://github.com/open-zaak/open-zaak-market-consultation) |
+The codebase SHOULD be in use by multiple parties. | yes |
+The roadmap SHOULD be influenced by the needs of multiple parties. | yes | [Dimpact](https://www.dimpact.nl/openzaak), [market consultation](https://github.com/open-zaak/open-zaak-market-consultation)
 Configuration SHOULD be used to make code adapt to context specific needs. |  |
-The codebase SHOULD include a machine readable metadata description, for example in a [publiccode.yml](https://github.com/publiccodeyml/publiccode.yml) file. | yes | [publiccode.yml](https://github.com/open-zaak/open-zaak/blob/master/publiccode.yaml)
+The codebase SHOULD be localizable. |  |
 Code and its documentation SHOULD NOT contain situation-specific information. | yes | Some GCloud examples but nothing required; no credentials and documentation suggests using secret generators
+Codebase modules SHOULD be documented in such a way as to enable reuse in codebases in other contexts. |  | <!-- need to investigate -->
```

Comment:
API modules/classes are currently stored in the general purpose `utils` package: https://github.com/open-zaak/open-zaak/tree/main/src/openzaak/utils

There are plans to move this into `openzaak.api` though. Typically they are extensions on top of the django-restframework package for the API implementations.

We follow similar patterns with admin UI functionality, re-usable validators etc.

Reply:
As discussed in our call, the documentation around this could be more like it is in Forms.